contracts/drafts/Strings.sol

```diff
@@ -0,0 +1,53 @@
+pragma solidity ^0.5.0;
+
+/**
+ * @title Strings
+ * @dev String operations.
+ * via OraclizeAPI - MIT licence
+ * https://github.com/oraclize/ethereum-api/blob/b42146b063c7d6ee1358846c198246239e9360e8/oraclizeAPI_0.4.25.sol
+ */
+library Strings {
+    /**
+     * @dev Concatenates two strings.
+     */
+    function concatenate(string memory a, string memory b) internal pure returns (string memory concatenatedString) {
+        bytes memory bytesA = bytes(a);
+        bytes memory bytesB = bytes(b);
+        string memory concatenatedAB = new string(bytesA.length + bytesB.length);
```

Comment:
I think this is wrong, since `string` users UTF-8: the byte arrays may be longer than the string. We should create a new `bytes` variable, and then convert it to `string` on return.

Reply:
I came across a different solution for concatenate which looks ok using abi.encodePacked

https://ethereum.stackexchange.com/a/56337/53724
https://github.com/ethereum/solidity/issues/4704

What do you think?